device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C28S4/BALANCED/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,52 @@
+{%- set default_cable = '300m' %}

```

Comment:
please copy buffers.json.j2 for this new SKU

Reply:
do you mean another file?